src/hardware/serialport/softmodem.cpp

```diff
@@ -81,7 +82,7 @@ bool MODEM_ReadPhonebook(const std::string &filename) {
 
 		if (!(iss >> phone >> address)) {
 			LOG_MSG("SERIAL: Phonebook skipped a bad line in %s.",
```

Comment:
Pre-existing issue; can you please enclose filenames in log messages with single-quotes? (so this will become `'%s'`, plus in this case I'd get rid of the period at the end because it could be confusing when the messages ends with a filename or a parameter)